### DISCUSSION TITLE
it's showing like I've paid but my subscription is not paid

### DISCUSSION THREAD

**Discussion by ghost** [community/community#103251] [Copilot Conversations]
### Select Topic Area

Bug

### Body

after entering my card details and finishing the payment I see the page below but after that my bank balance is the same and my account is not paid !!
![image_1110](https://github.com/community/community/assets/63903814/f3286dec-5602-4a67-a23e-11d55ce73735)

**Comment by Akash1134** [comment]
Hi @abdoo9 ,

There could be can issue with the payment or your account. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
